boards/arm/mm_swiftio/pinmux.c

```diff
@@ -20,30 +20,30 @@ static gpio_pin_config_t enet_gpio_config = {
 #if DT_NODE_HAS_STATUS(DT_NODELABEL(usdhc1), okay) && CONFIG_DISK_ACCESS_USDHC1
 
 /*Drive Strength Field: R0(260 Ohm @ 3.3V, 150 Ohm@1.8V, 240 Ohm for DDR)
- *Speed Field: medium(100MHz)
- *Open Drain Enable Field: Open Drain Disabled
- *Pull / Keep Enable Field: Pull/Keeper Enabled
- *Pull / Keep Select Field: Pull
- *Pull Up / Down Config. Field: 47K Ohm Pull Up
- *Hyst. Enable Field: Hysteresis Enabled.
+ * Speed Field: medium(100MHz)
+ * Open Drain Enable Field: Open Drain Disabled
+ * Pull / Keep Enable Field: Pull/Keeper Enabled
+ * Pull / Keep Select Field: Pull
+ * Pull Up / Down Config. Field: 47K Ohm Pull Up
+ * Hyst. Enable Field: Hysteresis Enabled.
  */
 
 static void mm_swiftio_usdhc_pinmux(
 	uint16_t nusdhc, bool init,
 	uint32_t speed, uint32_t strength)
 {
 	uint32_t cmd_data = IOMUXC_SW_PAD_CTL_PAD_SPEED(speed) |
-				IOMUXC_SW_PAD_CTL_PAD_SRE_MASK |
```

Comment:
Please move the unrelated whitespace changes to a separate commit

Reply:
Thanks for review, Fixed